src/_guides/libraries/library-tour.md

```diff
@@ -1293,6 +1293,60 @@ await Future.wait([
 print('Done with all the long steps!');
 ```
 
+`Future.wait()` returns a future which will complete once all the provided
+futures have completed, either with their results, or with an error if any of
+the provided futures fail. 
+
+#### Handling errors for multiple futures
+
+You can also wait for parallel operations on an [iterable]({{site.dart-api}}/{{site.data.pkg-vers.SDK.channel}}/dart-async/FutureIterable/wait.html)
+or [record]({{site.dart-api}}/{{site.data.pkg-vers.SDK.channel}}/dart-async/FutureRecord2/wait.html)
+of futures.
+
+These extensions return a future with the resulting values of the provided futures.
+Unlike `Future.wait()`, if any future in the collection completes with an error,
+`wait` completes with a [`ParallelWaitError`][] that allows the caller to
+handle individual errors and dispose successful results if necessary:
+
+```dart
+void main() async {
+  Future<bool> delete() async =>  ...
+  Future<bool> copy() async =>  ...
+  Future<bool> causesError() async =>  ...
+
+  try {
+    await [delete(), copy(), causesError()].wait;    // List of futures
+    Expect.fail("Didn't throw");
+  } on ParallelWaitError<List<bool?>, List<AsyncError?>> catch (e) {   
+    print(e.values[0]);    // Prints result of future
+    print(e.values[1]);    // Prints result of future
+    print(e.values[2]);    // Prints null
+
+    print(e.errors[0]);    // Prints null
+    print(e.errors[1]);    // Prints null
+    print(e.errors[2]);    // Preints error
+  }
+}
+```
+
+Waiting on a record of futures enables the same functionality, with the additional 
+feature that the futures can be of different types:
+
+```dart
+void main() async {
+  Future<int>    delete() async =>  ...
+  Future<String> copy() async =>  ...
+  Future<bool>   causesError() async =>  ...
+
+  try {
+    await (delete(), copy(), causesError()).wait;    // Record of futures
```

Comment:
Again, consider capturing the values, even if it's just:
```dart
  (int, String, bool) result = await (delete(), copy(), errorResult()).wait;
```

The fact that record-`.wait` allows different types of results 
is it's biggest feature, so it bears highlighting.

Reply:
Similar to my last question, is the `Expect.fail()` line right below necessary? I know I added this part, but now I'd like to get anything that's not absolutely necessary out for brevity. 

I was hoping the `errorResult()` future would imply that the result is expected to fail, just for the sake of the example. And I'm hoping if they want to learn more they'll go to the API docs for more complete examples (assuming that [the test samples](https://dart-review.googlesource.com/c/sdk/+/288903/12/tests/lib/async/future_record_extension_test.dart) show up in the api docs somewhere, and if they don't, they really should!)

Reply:
This is example code, possibly the first/only example code someone sees of `await (someRecord).wait`.

Unlike awaiting a list, the only reason to await a record of futures is if you *need* the result values. Otherwise you could just have made a `List<Future<void>>` instead of a record.

So I think the first example someone sees of `await (someRecord).wait` should show someone who *wants* to use the result values.

But that comes back to the purpose of the example:
* Is it to show an example of `await (someRecord).wait`, or
* Is it to show how `await (someRecord).wait` differs from ... something? `Future.wait`?

The latter seems too context dependent. If someone reads this example first, they won't have the context needed. They will read it as an example of using `await (someRecord).wait`, so it should be a *good* example of that first, and then something which can also be compared to other features second.

IMO. :)

Reply:
That's a _really_ good point. I was thinking that it'd be most useful to users, for some reason, to just see the bare minimum of how to set up using `wait` on a record. I didn't think of showing an actual useful use case. Thanks so much for explaining!

Reply:
> the only reason to await a record of futures is if you need the result values.

I added this description on top of the example, and showed some arbitrary "use" of the result values